import_flow_events.py

```diff
@@ -289,7 +301,6 @@ def import_events(force_reload=False):
     for key in b.list(prefix=EVENTS_PREFIX):
         filename = os.path.basename(key.name)
         day = "-".join(filename[:-4].split("-")[1:])
-        print day
```

Comment:
This little guy outlived his usefulness a long time ago and just wastes precious newlines in my terminal buffer now.